ports/servoshell/egl/ohos/simpleservo.rs

```diff
@@ -53,6 +53,13 @@ pub fn init(
     );
     debug!("Servo commandline args: {:?}", args);
 
+    if crate::prefs::OVERRIDE_DEFAULT_PREFS_DIR
+        .set(resource_dir)
+        .is_err()
```

Comment:
Instead of if and .is_err() you could just use 
.inspect_err() and print the error there